app/Livewire/Settings/Updates.php

```
@@ -0,0 +1,101 @@
+    #[Validate('string')]
+    public string $auto_update_frequency;
+
+    #[Validate('string|required')]
+    public string $update_check_frequency;
+
+    #[Validate('boolean')]
+    public bool $is_auto_update_enabled;
+
+    public function mount()
+    {
+        $this->server = Server::findOrFail(0);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Server' in method 'mount'. (Clean Code Rules)

(StaticAccess)